packages/errors/src/errors.ts

```diff
@@ -86,7 +86,7 @@ export const AllCypressErrors = {
   },
   CHROME_WEB_SECURITY_NOT_SUPPORTED: (browser: string) => {
     return errTemplate`\
-        Your project has set the configuration option: ${fmt.highlight(`chromeWebSecurity`)} to ${fmt.highlightTertiary(`false`)}
+        Your project has set the configuration option: \`chromeWebSecurity\` to \`false\`.
```

Comment:
Problem is that this is displayed in the browser hover tooltip, which won't render the ANSI escape sequences. Rather than implement that I chose to change this message to use plaintext.

Reply:
Ah right, okay sounds good